networking/about-contour-operator.adoc

```diff
@@ -0,0 +1,22 @@
+[id="about-contour-operator"]
+= About the Contour Operator
+include::modules/common-attributes.adoc[]
+:context: about-contour-operator
+
+toc::[]
+
+//Dev Preview. OKD only.
+
+The Contour Operator can be installed on {product-title} to manage Contour, an Ingress Controller that supports Gateway API, which exposes Kubernetes resources such as services to external consumers.
+
+include::modules/nw-install-contour-operator.adoc[leveloffset=+1]
+
+include::modules/nw-install-config-gateway-api.adoc[leveloffset=+1]
+
+[id="about-contour-operator-additional-resources"]
+== Additional resources
+
+For additional details, see:
+
+* link:https://gateway-api.sigs.k8s.io/[What is the Gateway API]
```

Comment:
Do you need this link here, where it is used in two other places in the assembly?